sponsor-dapp-v2/src/views/Withdraw.js

```diff
@@ -73,8 +93,10 @@ class Withdraw extends Component {
 
               <div className="popup__head-entry">
                 <p>
-                  <strong>Your facility has a 110% collateralization requirement.</strong> You can withdraw collateral
-                  from your facility as long as you maintain this requirement.{" "}
+                  <strong>
+                    Your facility has a {fromWei(collateralizationRequirement)}% collateralization requirement.
+                  </strong>{" "}
```

Comment:
Just for my understanding, what does this `{" "}` element do?

Reply:
I _think_ it just puts a space haha. That's what prettier puts in when something in your JSX is too long of a line.

Reply:
ooooooh, this makes sense. I've seen this in yours and Roz's code, and I had no idea where y'all were getting that syntax - thanks.